app.py

```diff
@@ -13,6 +13,7 @@
 from leaderboard import SUPPORTED_TRANSLATION_LANGUAGES
 from model import check_models
 from model import supported_models
+from rate_limit import set_token
```

Comment:
I wonder `rate_limit` module is a right place to have `set_token`.

Reply:
I understand what you mean. at the same time the limiter limits rate based on tokens so set_token should be in rate_limit
would it be better if we rename `set_token` to `set_rate_limit_token`?

Reply:
`import rate_limit` and `rate_limit.set_token` seems better.
reasons: https://google.github.io/styleguide/pyguide.html#22-imports`import rate_limit`